src/components/NavBar/NavigationBar.tsx

```diff
@@ -11,6 +11,7 @@ import LightDarkSlider from "@/components/NavBar/LightDarkSlider";
 import SignOutButton from "@/components/NavBar/SignOutButton";
 import NavBarButton from "@/components/Buttons/NavBarButton";
 import { usePathname } from "next/navigation";
+import { RoleUpdateContext, rolesToHiddenPages } from "@/app/roles";
 
 export const NavBarHeight = "4rem";
 
```

Comment:
I noticed that we still have background-color: ${(props) => props.theme.primaryBackgroundColor} in this file. I know its not in your ticket, but can you quickly refactor this?